### PR TITLE
docs: Add a function example with ggraph to the README

### PR DESCRIPTION
**Pull request by jvfe**
- Adds an example to the README using the easylayout function, per suggestion by @diegomscoelho

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,27 @@ igraph::V(g)$color <- sample(rainbow(5), number_of_vertices, replace = TRUE)
 plot(g, layout = easylayout, vertex.label = NA, margin = 0)
 ```
 
+You can also run easylayout as a function and attribute the results to a
+variable, containing the layout itself:
```
Comment (daniloimparato) [inline README.md:58]:
```suggestion
You can also run easylayout as a standalone function, attributing its output
to a variable in your R environment. The output is just a 2-column matrix:
```

File: README.md [hunk 2]
```diff
@@ -54,6 +54,27 @@ igraph::V(g)$color <- sample(rainbow(5), number_of_vertices, replace = TRUE)
...
+
+``` r
+layout <- easylayout(g)
+```
+
+Once you achieve your desired layout, retrieve the results back to your
+R environment and use a plotting package to display the resulting
+network. In the example below, we set the x and y values to node
+attributes and plot the network using ggraph.
```
Comment (daniloimparato) [inline README.md:67]:
```suggestion
Once you store the final coordinates to a variable, use any plotting package to
display the network. In the example below, we plot the network using ggraph.
```

File: README.md [hunk 3]
```diff
@@ -54,6 +54,27 @@ igraph::V(g)$color <- sample(rainbow(5), number_of_vertices, replace = TRUE)
...
+
+``` r
+V(g)$x <- layout[, 1]
+V(g)$y <- layout[, 2]
+
+ggraph::ggraph(g, layout = "manual", x = V(g)$x, y = V(g)$y) +
+  ggraph::geom_edge_link() +
+  ggraph::geom_node_point()
```
Comment (daniloimparato) [inline README.md:75]:
```suggestion
ggraph::ggraph(g, layout = layout) +
  ggraph::geom_edge_link() +
  ggraph::geom_node_point(aes(color = color)) + 
  coord_fixed() +
  theme_void()
```

**Review by daniloimparato — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jvfe) [reply README.md:75]: bunitin

**Review by daniloimparato — APPROVED** [review]

**Pull request merged**